### DISCUSSION TITLE
Search dataset usage

### DISCUSSION THREAD

**Discussion by llrs** [r-universe-org/help#546] [Ideas]
Would it be possible to filter or search datasets? Say one is at [pharmaverse](https://pharmaverse.r-universe.dev/datasets) and want only datasets from teal* packages. I can do a simple web search but if I want to know if a given dataset is used in other packages, is there a way to do that with r-universe?

**Comment by jeroen** [comment]
There is a special page listing all the datesets, that also has a filterbox: https://r-universe.dev/datasets
Does that help?

**Reply by llrs** [reply]
Ah, I forgot about this special page. Thanks for the reminder. This helps finding how many are on a given package but not to find where are they used.
